product_docs/docs/pgd/4/rel_notes/pgd_4.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.2"
+navTitle: "Version 4.3.2"
+---
+
+EDB Postgres Distributed version 4.3.2 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.2   | Feature         | Fix a bug in handling of memory context in presence of triggers |
+| BDR       | 4.3.2   | Feature         | Add some simplistic memory tracking util |
+| BDR       | 4.3.2   | Feature         | Add a test case for bulkload with triggers on downstream |
+| BDR       | 4.3.2   | Feature         | Prevent overflow of EagerTxnConfig::node_idss array |
+| BDR       | 4.3.2   | Feature         | Add upgrade path from 3.7.22 |
+| BDR       | 4.3.2   | Feature         | Fixup bdr.sequence_alloc upgrade script |
+| BDR       | 4.3.2   | Feature         | Check for malformed connection string |
```

Comment:
I would recommend mentioning user visible change here. `bdr.create_node()` will fail on malformed connection string. Maybe also reference the customer ticket.